copy-from-s3bucket.js

```diff
@@ -0,0 +1,106 @@
+require('dotenv').config();
+const fs = require('fs');
+const path = require('path');
+const AWS = require('aws-sdk');
+
+const {
+  AWS_ACCESS_KEY_ID,
+  AWS_SECRET_ACCESS_KEY,
+  AWS_REGION,
+  S3_BUCKET_NAME,
+} = process.env;
+
+// Configure AWS S3
+const s3 = new AWS.S3({
+  accessKeyId: AWS_ACCESS_KEY_ID,
+  secretAccessKey: AWS_SECRET_ACCESS_KEY,
+  region: AWS_REGION
+});
+
+async function listFiles() {
+  const params = {
+    Bucket: S3_BUCKET_NAME
+  };
+
+  try {
+    const data = await s3.listObjectsV2(params).promise();
+    return data.Contents.map(item => item.Key);
+  } catch (error) {
+    console.error('Error listing files:', error.message);
+    return [];
+  }
+}
+
+async function downloadFile(fullFileName) {
+  // Skip directories in S3 (by checking for a trailing slash)
+  if (fullFileName.endsWith('/')) {
+    return;
+  }
+
+  let targetFolder = '';
+  let companyDir = '';
+  let versionDir = '';
+
+  // Extract version and companyName from the path
+  const pathParts = fullFileName.split('/');
+
+  const fileName = pathParts.pop();
+  const versionNumber = pathParts.pop();
+  const companyName = pathParts.pop();
+
+  if (fullFileName.startsWith('test/fixtures/')) {
```

Comment:
If you run this script in `copy-resources.sh` before copying across the schemas and fixtures, you can remove this logic and just add the files directly to `schemas` and `test/fixtures`. The `copy-resources` script should then take care of the rest.